api/app/Jobs/GenerateUserFile.php

```diff
@@ -34,7 +34,7 @@ public function handle(): void
         } catch (\Throwable $e) {
             // Notify the user something went wrong
             $this->user->notify(new UserFileGenerationError($this->generator->getFileName()));
-            Log::error('Error generating file: '.$e->getMessage().' '.$e->getFile().':'.$e->getLine());
+            Log::error($e);
```

Comment:
I hope this works for an error log... Do we need to say at what step the error occurred or do you think the stack trace makes it obvious?